pkg/rpcutil/rpc.go

```diff
@@ -24,15 +24,28 @@ import (
 	"time"
 
 	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
 )
 
+// grpcDialDefaultTimeout is the default timeout for initial gRPC dial
+const grpcDialDefaultTimeout = 2 * time.Second
```

Comment:
It is not, but it's subject of ongoing research and should be left here for now.